src/shared_utilities/DataHandling.jl

```diff
@@ -0,0 +1,246 @@
+module DataHandling
+
+using Dates
+
+import ClimaCore
+
+import ..FileReader: NCFileReader
+import ..Regridder
+import ..Regridder:
+    AbstractRegridder, TempestRegridder, regrid, AVAILABLE_REGRIDDERS
+
+"""
+
+Note: It is best to have one file per variable with all the temporal data.
```

Comment:
this may be a little inconvenient/ add a step - all the reanalysis data is in one file to start with.